Balance

berin-hadziabdic-dev opened the pull request:
Declare income PR to the correct repo fork. 
This PR has all of our tests, both positive cases and negative cases.

Grantimatter committed:
initial commit

Grantimatter committed:
Initial Angular Setup

Grantimatter committed:
Initial Angular Setup

Grantimatter committed:
Initial Angular Setup

Grantimatter committed:
Initial Angular Setup

Grantimatter committed:
created feature module

Grantimatter committed:
created feature module

Grantimatter committed:
created feature module

jlrjr98 committed:
package-lock.json update

Commit:
Created project structure for authentication team

jlrjr98 committed:
config of .json files

Commit:
add login form

Grantimatter committed:
Added empty service and component to feature module

Grantimatter committed:
Added empty service and component to feature module

Grantimatter committed:
Added empty service and component to feature module

Grantimatter committed:
Added empty service and component to feature module

htl43 committed:
initial for balance component

htl43 committed:
initial for balance component

Grantimatter committed:
Added a test for createChart()

Grantimatter committed:
Added a test for createChart()

Commit:
Refactored code to comply with design principles

jlrjr98 committed:
Add calculateBalanceChart method and test

jlrjr98 committed:
Added calculateBalanceChart method and test

Grantimatter committed:
Created and updated tests

Grantimatter committed:
Created and updated tests

Grantimatter committed:
Merge branch 'balance-projection' of https://github.com/personal-finance-app1/personal-finance-front into balance-projection

logriffith committed:
worked on getAccount test

logriffith committed:
getting Grant's changes

logriffith committed:
worked on getAccount test

Grantimatter committed:
Merged remote and general cleanup

Grantimatter committed:
Merged remote and general cleanup

Grantimatter committed:
Merge branch 'balance-projection' of https://github.com/personal-finance-app1/personal-finance-front into balance-projection

Grantimatter committed:
Declared getAccount() unit test as pending

Grantimatter committed:
rebase dev to main

Berin-Hadziabdic committed:
Added stub functionality for services and component.

I created the interface for our local balance service, and I created the interface for our balance component. There is no implementation as we are doing TDD. This should enable our other team members to begin work on their assigned tasks.

Berin-Hadziabdic committed:
Added stub functionality for services and component.

I created the interface for our local balance service, and I created the interface for our balance component. There is no implementation as we are doing TDD. This should enable our other team members to begin work on their assigned tasks.

Berin-Hadziabdic committed:
Fixed giant whitespace gap.

Berin-Hadziabdic committed:
Fixed giant whitespace gap.

Commit:
updated auth module

Grantimatter committed:
Rebase dev with main

Grantimatter committed:
Rebase dev with main

Grantimatter committed:
Merge branch 'dev' of https://github.com/personal-finance-app1/personal-finance-front into dev

logriffith committed:
modified getAccount tests

Grantimatter committed:
rebase balance-projection with dev

Grantimatter committed:
removed get balance test

Grantimatter committed:
Merge branch 'dev' into balance-projection

jlrjr98 committed:
merge conflict fix before pull

logriffith committed:
working on negative getAccount test

logriffith committed:
working on negative getAccount test

Grantimatter committed:
Finalized first draft of jasmine unit tests

Grantimatter committed:
Added account subject in environment.ts

Grantimatter committed:
Added account subject to environment.ts

Grantimatter committed:
Added account subject to environment.ts

fendtg4 committed:
adding tests for balance service

htl43 committed:
adding template for balance and material

htl43 committed:
adding template for balance and material

htl43 committed:
commit for adding balance template component abd material module

htl43 committed:
commit for adding balance template component abd material module

htl43 committed:
commit for adding balance template component abd material module

htl43 committed:
remove method function

htl43 committed:
remove method function

Rebasing from dev to balance.

Grantimatter committed:
Created an Account model and added a Subject<Account> to the environment to act as our dynamic account object.

Grantimatter committed:
Created an Account model and added a Subject<Account> to the environment to act as our dynamic account object.

Berin-Hadziabdic committed:
Mergin local updates to remote branch.

Berin-Hadziabdic committed:
Mergin local updates to remote branch.

Berin-Hadziabdic committed:
Merge branch 'balance' of https://github.com/personal-finance-app1/personal-finance-front into balance

I merged my local, out of sync branch, with the remotes new changes and updates.

Grantimatter committed:
Added firebase credentials .json file to gitignore just in case.

Grantimatter committed:
Merge branch 'dev' of https://github.com/personal-finance-app1/personal-finance-front into dev

# Conflicts:
#	.gitignore
#	angular.json
#	package-lock.json
#	package.json
#	src/app/app.component.html
#	src/app/app.module.ts

Grantimatter committed:
Merge branch 'balance-projection' into dev

# Conflicts:
#	package-lock.json
#	src/environments/environment.ts

Berin-Hadziabdic committed:
Added BalanceComponent tests

I added six tests to test our balance our balance component. They have been updated to be compliant with the project's testing syntax.

Berin-Hadziabdic committed:
Added BalanceComponent tests

I added six tests to test our balance our balance component. They have been updated to be compliant with the project's testing syntax.

Grantimatter committed:
Added valid constructor for account test

Grantimatter committed:
Corrected import for account

fendtg4 committed:
adding tests for balance service

fendtg4 committed:
adding tests for balance service

Commit:
Added unit-testing to AuthService

Commit:
Confirming all AuthService changes are pushed

Commit:
Syncing auth with dev

Berin-Hadziabdic committed:
Preparing for PR.

Made minor changes to prepare for PR.

Berin-Hadziabdic committed:
Preparing for PR.

Made minor changes to prepare for PR.

Berin-Hadziabdic committed:
Made our tests  project style compliant.

Berin-Hadziabdic committed:
Made our tests  project style compliant.

Berin-Hadziabdic committed:
Changes to package.json

Berin-Hadziabdic committed:
Changes to package.json

Berin-Hadziabdic committed:
Added comma

I am updating my local repo so I can rebase our branch. I added a comma in app.module.ts to fix a minor syntax error.

Berin-Hadziabdic committed:
syncing local dev with remote dev

Berin-Hadziabdic committed:
Rebasing with dev.

I am rebasing our branch with dev.

Berin-Hadziabdic committed:
Fixing rebase merge conflict.

I performed a rebase but there were a few minor whitespace related merge conflicts to be resolved from the remote balance branch.

GattieTime commented on debug.log:
```diff
@@ -0,0 +1,96 @@
+[0113/153500.789:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
please add this file to the gitignore and remove it from the branch.

GattieTime commented on package.json:
```diff
@@ -16,19 +16,22 @@
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1100.5",
-    "@angular/cli": "~11.0.5",
+    "@angular-devkit/build-angular": "^0.1100.7",
```
updates

GattieTime commented on src/app/app.module.ts:
```diff
@@ -1,22 +1,37 @@
   ],
   providers: [],
-  bootstrap: [AppComponent]
+  bootstrap: [AppComponent],
```
why?

GattieTime commented on src/app/app.module.ts:
```diff
@@ -1,22 +1,37 @@
+  bootstrap: [AppComponent],
 })
-export class AppModule { }
+export class AppModule {}
```
Why?

GattieTime commented on src/app/modules/auth/auth.module.ts:
```diff
@@ -0,0 +1,81 @@
+import { NgModule } from '@angular/core';
```
These files are duplicated from another PR.

Berin-Hadziabdic committed:
Fixing PR incompatabilities.

debug.log has been cleared out and .gitignore has been synced up to include debug.log

Berin-Hadziabdic committed:
I removed debug.log from our branch.

Grantimatter committed:
Updated package.json to match main

Berin-Hadziabdic committed:
Removed old package-lock.json.

I removed our outdated package-lock.json.

Berin-Hadziabdic committed:
Removing auto generated package-lock.json

Our branch had an outdated package-lock.json. I removed it to make merging with main easier.

Berin-Hadziabdic committed:
Merging main into Balance.

Berin-Hadziabdic committed:
Merging balance with main.

berin-hadziabdic-dev committed:
Updating

White space related fixes to resolve mc.

berin-hadziabdic-dev committed:
Updated cli/devkit versioning conflict.

berin-hadziabdic-dev committed:
Added /n to first line of file to resolve mc.

berin-hadziabdic-dev committed:
white space fixes.

berin-hadziabdic-dev committed:
Whitespace fix.

berin-hadziabdic-dev committed:
Whitespace updates.

Berin-Hadziabdic committed:
Updated pkg-lock.json

berin-hadziabdic-dev committed:
white space.

berin-hadziabdic-dev committed:
Auto formatted comma removal.

berin-hadziabdic-dev committed:
Auto format update.

GattieTime reviewed (approved):
Approved.
